src/commands/Utility/spotify.js

```diff
@@ -1,95 +1,136 @@
+const RichEmbed = require('discord.js').RichEmbed;
+const SpotifyWebApi = require('spotify-web-api-node');
+const SpotifyUriParser = require('spotify-uri');
 const got = require('got');
+const spotifyApi = new SpotifyWebApi();
+
+exports.run = async function (bot, msg, args) {
```

Comment:
Please use lambda notation (`(...) => { ... }`) instead of function notation (`function (...) { ... }`) for the sake of consistency in code style.

Reply:
Please edit the _base.js then, since i took it from there :P